cli/provision.go

```diff
@@ -124,7 +124,7 @@ var cmdProvision = []cobra.Command{
 				Email:    un,
 				Password: "12345678",
 			}
-			if _, err := sdk.CreateUser(user); err != nil {
+			if _, err := sdk.CreateUser("t", user); err != nil {
```

Comment:
What's this `"t"` string?

Reply:
CreateUser takes a token to verify the user. If a token is an empty string, validation of the [`createUserReq`](https://github.com/buraksekili/mainflux/blob/ee866f1fc7dd5a2bfa54651f1e6a0729d2ca8ea5/users/api/endpoint.go#L18) will be failed. Therefore, `"t"` is an arbitrary value.

Reply:
I would prefer than a defined constant string with some self-describing value - like `"foobar"` and a comment that this is to avoid validation fail.

BTW, can we maybe permit empty strings, and fail validation only when policies do not allow self-registration? That would make more sense and make it cleaner.

@mteodor @nmarcetic what do you think?